src/module-elasticsuite-core/Model/Search/Request/RelevanceConfig/Structure/Data.php

```diff
@@ -0,0 +1,26 @@
+<?php
+/**
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade Smile ElasticSuite to newer
+ * versions in the future.
+ *
+ * @category  Smile
+ * @package   Smile\ElasticsuiteCore
+ * @author    Thomas Klein <thomasklein876@gmail.com>
+ * @copyright 2020 Smile
+ * @license   Open Software License ("OSL") v. 3.0
+ */
+declare(strict_types=1);
+
+namespace Smile\ElasticsuiteCore\Model\Search\Request\RelevanceConfig\Structure;
+
+use Magento\Framework\Config\Data\Scoped;
+
+class Data extends Scoped
+{
+    public function merge(array $config): void
```

Comment:
Please add PHPdoc for the function.
You can use 
```
    /**
     * {@inheritDoc}
     */
```

Reply:
You should update your phpcs rules, brackets over @inheritDoc shouldn't be used and this tag has only interest if the description or arguments is updated in the overridden method.
The method is self descriptive and already signed.

Reply:
Maybe. But we use extensively {@inheritDoc} as it is and the rule about every function being documented is not going away anytime soon.